neo/SmartContract/Native/Votes/Interface/IMultiVoteModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Neo.SmartContract.Native.Votes.Interface
+{
+    interface IMultiVoteModel
+    {
+        //TODO: details for interface
+        int[,] CalculateVote(List<CalculatedMultiVote> votes);
```

Comment:
int[,] ?

Reply:
Let's create a matrix here, @Lichen9618.
Define like?

Matrix[int] of type int[,], it will be easier I think.